packages/modules-sdk/src/definitions.ts

```diff
@@ -118,6 +121,20 @@ export const ModulesDefinition: { [key: string | Modules]: ModuleDefinition } =
         resources: MODULE_RESOURCE_TYPE.SHARED,
       },
     },
+    [Modules.PROMOTION]: {
+      key: Modules.PROMOTION,
+      registrationName: ModuleRegistrationName.PROMOTION,
+      defaultPackage: false,
+      label: "promotionModuleService",
```

Comment:
todo: I think that we should use `ModuleRegistrationName.PROMOTION` same for the other ones wdyt? since the label is the registration name